protocols/tests/test_migration/base_test_migration.py

```diff
@@ -29,4 +29,4 @@ def _check_non_empty_fields(self, instance, exclusions=[]):
                         self._check_non_empty_fields(attribute, exclusions)
 
     def _validate(self, instance):
-        self.assertTrue(instance.validate(instance.toJsonDict()))
+        self.assertTrue(instance.validate(instance.toJsonDict(), verbose=True))
```

Comment:
passing `verbose=True` is only useful if you want to access the `messages` attribute of the result. The way it's used here, it's the same result whether you pass it or not.

Reply:
Interesting that you comment that because I spotted a case where verbose validation was passing, while non verbose validation was actually failing! I would need to look at some data and try to reproduce.